RATapi/project.py

```diff
@@ -529,7 +529,9 @@ def set_absorption(self) -> "Project":
         handle = self.layers._class_handle.__name__
         if self.absorption and handle == "Layer":
             for layer in self.layers:
-                layer_list.append(RATapi.models.AbsorptionLayer(**layer.model_dump()))
+                layer_params = layer.model_dump()
+                layer_params["SLD_imaginary"] = ""
```

Comment:
Is this to account for the possibility of `SLD_imaginary` being empty?

Reply:
No, this code is for when you change `project.absorption` from `False` to `True`. At that point you need to switch the normal layer to the absorption layer, so the (now required) `SLD_imaginary` is set to be empty. It's up to the user to fill it in at some point.